src/API/Rest/DicomWebConnectionDetails.cs

```diff
@@ -25,6 +25,23 @@ namespace Nvidia.Clara.DicomAdapter.API.Rest
     /// </summary>
     public class DicomWebConnectionDetails
     {
+        /// <summary>
+        /// Gets or sets BaseUri.
+        /// This property is intentionally private and maps to Uri to reduce exposed properties.
+        /// Tip: Placing a trailing forward slash (/) may fix some 404 errors.
+        /// </summary>
+        /// <value></value>
+        [JsonProperty("baseUri")]
+        private string BaseUri { set { Uri = value; } }
+
+        /// <summary>
+        /// Gets or sets ApiKey.
+        /// This property is intentionally private and maps to AuthId to reduce exposed properties.
+        /// </summary>
+        /// <value></value>
+        [JsonProperty("apiKey")]
+        private string ApiKey { set { AuthId = value; } }
```

Comment:
is the connection encrypted? best to not send/receive credentials when the connection isn't encrypted.

Reply:
Good call..  I will put a note in the user guide.